tests/units/test_runner.py

```diff
@@ -7,59 +7,44 @@
 
 import logging
 import resource
+import sys
 from pathlib import Path
 from unittest.mock import patch
 
 import pytest
 
-from anta import logger
 from anta.catalog import AntaCatalog
 from anta.inventory import AntaInventory
 from anta.result_manager import ResultManager
 from anta.runner import adjust_rlimit_nofile, main, prepare_tests
 
-from .test_models import FakeTest
+from .test_models import FakeTest, FakeTestWithMissingTest
 
 DATA_DIR: Path = Path(__file__).parent.parent.resolve() / "data"
 FAKE_CATALOG: AntaCatalog = AntaCatalog.from_list([(FakeTest, None)])
 
 
-async def test_runner_empty_tests(caplog: pytest.LogCaptureFixture, test_inventory: AntaInventory) -> None:
-    """Test that when the list of tests is empty, a log is raised.
-
-    caplog is the pytest fixture to capture logs
-    test_inventory is a fixture that gives a default inventory for tests
-    """
-    logger.setup_logging(logger.Log.INFO)
```

Comment:
how come this is not needed anymore?

Reply:
Was this needed in the first place ?

Reply:
seems like it was copied around not sure :) if it passes without means we still are getting the right error message. Probably some refactor made it useless